recipes/malt/meta.yaml

```diff
@@ -0,0 +1,32 @@
+# This file created by conda-build 3.15.1
+# meta.yaml template originally from:
+# /opt/recipe, last modified Fri Oct 26 10:28:41 2018
+# ------------------------------------------------
+
+package:
+    name: malt
+    version: 0.41
+source:
+    md5: 1e7b3d126e3f761e7d09ddf0029a3dff
+    url: https://ab.inf.uni-tuebingen.de/data/software/malt/download/MALT_unix_0_4_1.sh
+build:
+    noarch: generic
+    number: '0'
+    string: '0'
```

Comment:
Please remove this line.

Reply:
Do you only refer to the last line string: '0 '? Because the rest is necessary to pass the lint test

Reply:
Just `string: '0'`. Please also change `number` to `number: 0`, since it should be an integer rather than a string.